src/CMakeLists.txt

```diff
@@ -19,6 +17,22 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-ESPRESSO_SCRIPTS="@ESPRESSO_SCRIPTS@"
-export ESPRESSO_SCRIPTS
-@ESPRESSO_BIN@ $*
+cmake_minimum_required(VERSION 2.6)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+configure_file(${CMAKE_SOURCE_DIR}/cmake/cmake_config.cmakein
```

Comment:
Don't need ` ${CMAKE_SOURCE_DIR}` here, `configure_file` takes a target path relative to the current source directory. So use `..` instead.